docs/getting-started.md

```diff
@@ -78,16 +80,27 @@ When you run this command, you'll be taken to the Azure login screen in a web br
 
 Next, let's provision:
 
+CLI:
 ```bash
 matcha provision
 ```
 
-Initially, Matcha will ask you a few questions about how you'd like your infrastructure to be set up. Specifically, it will ask for a _name_ for your infrastructure, a _region_ to deploy it to. Once these details are provided, Matcha will proceed to initialize a remote state manager and ask for a password. After that, it will go ahead of provision infrastructure.
+> Note: users have the choice of passing optional arguments representing the location, prefix, and password parameters by using '--location', '--prefix', or '--password'. For example; `--location uksouth --prefix test123 --password strong_password`. 
+
+API:
+```python
+import matcha_ml.core as matcha
+
+_ = matcha.provision(location="uksouth", prefix="test123", password="strong_password")
```

Comment:
Might be worth specifying the variable that's returned for clarity: `matcha_state: MatchaState = ...`

Reply:
Good idea: added.